fcrepo-jms/src/test/java/org/fcrepo/integration/jms/observer/AbstractJmsIT.java

```diff
@@ -135,6 +140,12 @@
 
     protected abstract Destination createDestination() throws JMSException;
 
+    @Before
+    public void setUp() {
+        setField(createResourceService, "referenceService", referenceService);
```

Comment:
It appears that setting the fields here is unnecessary. I believe the `setUp` method can be removed.